commander/src/bootstrapping/templates/lisk-template-ts/templates/plugin/src/app/plugins/plugin.ts

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright © 2020 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ */
+
+import { BasePlugin, PluginInfo } from 'lisk-framework';
```

Comment:
this should be imported from `lisk-sdk` as `lisk-framework` is not in dependencies